diff --color -Naur a/asciiquarium b/asciiquarium
--- a/asciiquarium	2021-07-12 16:03:57.554113739 -0400
+++ b/asciiquarium	2021-07-12 16:04:20.704867768 -0400
@@ -537,8 +537,8 @@
 	my $collisions = $fish->collisions();
 	foreach my $col_obj (@{$collisions}) {
 		if($col_obj->type eq 'teeth') {
-			add_splat($anim, $col_obj->position());
-			$fish->kill();
+			# add_splat($anim, $col_obj->position());
+			# $fish->kill();
 			last;
 		} elsif($col_obj->type eq 'hook_point') {
 			retract($col_obj);
@@ -633,7 +633,7 @@
 
 
 
-                                           cR
+                                           cW
  
                                           cWWWWWWWW
 
@@ -645,7 +645,7 @@
 
 
 
-        Rc
+        Wc
 
   WWWWWWWWc
 
