diff -ruN onedrive.orig/Makefile onedrive/Makefile
--- onedrive.orig/Makefile	2020-01-14 00:03:20.942532156 -0500
+++ onedrive/Makefile	2020-01-13 23:54:27.817403146 -0500
@@ -25,7 +25,7 @@
 	install -D onedrive $(DESTDIR)$(PREFIX)/bin/onedrive
 	install -D -m 644 onedrive.service $(DESTDIR)/usr/lib/systemd/user/onedrive.service
 
-onedrive: version $(SOURCES)
+onedrive: $(SOURCES)
 	$(DC) $(DFLAGS) $(SOURCES)
 
 onedrive.service:
@@ -33,7 +33,4 @@
 
 uninstall:
 	rm -f $(DESTDIR)$(PREFIX)/bin/onedrive
-	rm -f $(DESTDIR)/usr/lib/systemd/user/onedrive.service
-
-version: .git/HEAD .git/index
-	echo $(shell git describe --tags) >version
\ No newline at end of file
+	rm -f $(DESTDIR)/usr/lib/systemd/user/onedrive.service
\ No newline at end of file
diff -ruN onedrive.orig/src/main.d onedrive/src/main.d
--- onedrive.orig/src/main.d	2020-01-14 00:03:20.942532156 -0500
+++ onedrive/src/main.d	2020-01-13 23:54:16.011331692 -0500
@@ -59,7 +59,7 @@
 	stdout.setvbuf(0, _IONBF);
 
 	if (printVersion) {
-		std.stdio.write("onedrive ", import("version"));
+		std.stdio.write("onedrive v1.1.3");
 		return EXIT_SUCCESS;
 	}
 
