Common subdirectories: a/assets and b/assets
diff -u a/catppuccin.tmux b/catppuccin.tmux
--- a/catppuccin.tmux	1969-12-31 19:00:01.000000000 -0500
+++ b/catppuccin.tmux	2023-02-01 01:24:22.093180454 -0500
@@ -30,13 +30,7 @@
   local theme
   theme="$(get-tmux-option "@catppuccin_flavour" "mocha")"
 
-  # NOTE: Pulling in the selected theme by the theme that's being set as local
-  # variables.
-  sed -E 's/^(.+=)/local \1/' \
-      > "${PLUGIN_DIR}/catppuccin-selected-theme.tmuxtheme" \
-      < "${PLUGIN_DIR}/catppuccin-${theme}.tmuxtheme"
-
-  source "${PLUGIN_DIR}/catppuccin-selected-theme.tmuxtheme"
+  source "${PLUGIN_DIR}/catppuccin-${theme}.tmuxtheme"
 
   # status
   set status "on"
