diff --git a/lib/wallstreet/w3m b/lib/wallstreet/w3m
index a53bbff..a825250 100755
--- a/lib/wallstreet/w3m
+++ b/lib/wallstreet/w3m
@@ -28,7 +28,7 @@ while true; do
 	cat $TMPDIR/finance | \
 		egrep "^[A-Z]+\s\s+.*[0-9]" | \
 		sed -e "s/^\[.*\]\s\+//" | \
-		/usr/lib/$PKG/breaks | \
+		"$(dirname "$0")/../../lib/$PKG/breaks" | \
 		perl -pe 's:-[0-9\.%]+[^\s\)]:\033[31;1m$&\033[30;0m:g' | \
 		perl -pe 's:\+[0-9\.%]+[^\s\)]:\033[32;1m$&\033[30;0m:g' | \
 		perl -pe 's:\s[\(\s][0-9\.]+%[\)\s]:\033[32;1m$&\033[30;0m:g' | \
